_i18n/en.yml

```diff
@@ -71,6 +72,7 @@ navigation:
   resources: Resources
   about: About
   mrl: Research Lab
+  builtonmonero: Built On Monero
```

Comment:
I would just use `titles.builtonmonero`

Reply:
i had a quick look at this, logically we should be able to access the value with `{{ titles.builtonmonero }}` but apparently this is not possible, for some reason i would need to access it from another file e.g. ,moving all titles to adding a `titles.yml` in `_data/` which seems silly (translations). will look further when i can

Reply:
I did use the same key-pairs in different context before, so shouldn't be an issue. Also, for the engine doesn't make any difference where the string comes from.

Reply:
indeed, i have seen `{{ site.baseurl }}` being used. but that also does not work here strangely. `titles.builtonmonero` displays as this text `{{"titles.builtonmonero"=>nil}=>nil}` - so i can only assume the location of the file is the issue and the value must be accessed with something like site.titles .. will look at it again soon

Reply:
Any news?

Reply:
No good news, just tried a few more attempts without success. I did find a similar question here https://stackoverflow.com/questions/42062883/jekyll-include-one-yaml-file-in-another-yaml-file but i was unable to get the example working. the OP seems to have moved the titles to a separate common file / wrote a plugin / and we may have to change the structure of the yaml slightly to make it work

Reply:
I'm not understanding the issue. I was suggesting to use `titles.builtonmonero` instead of creating `navigation.builtonmonero`, which would have the same content as the former. `titles.builtonmonero`  appears correctly in the front matter (`title: titles.builtonmonero`) and it's rendered properly. There is only an extra `navigation.builtonmonero` if i'm not missing something.

Reply:
huge misunderstanding on my end, apologies. for some reason i have been 100% certain that there is a loop which uses navigation: to generate the drop down tabs (a false memory after working on the hangouts page/forgetting that i hardcoded it already) - i have made the simple fix and will push soon